code/game/gamemodes/malfunction/malfunction.dm

```diff
@@ -48,7 +48,7 @@
 	if (malf_ai.len < required_enemies)
 		return 0
 	for(var/datum/mind/ai_mind in malf_ai)
-		ai_mind.assigned_role = "MODE"
+		ai_mind.assigned_role = "malfunctioning AI"

```

Comment:
I'm just nit-picking here but the "malfunctioning AI" should be "Malfunctioning AI". :P